products/urls.py

```diff
@@ -1,6 +1,6 @@
 from django.urls import path
-
-from .views      import RandomCategoryView
+from products.views      import CategoryListView
 
 urlpatterns = [
+    path('/main/<int:sub_category_id>',CategoryListView.as_view())
```

Comment:
페이지 단위로 view를 작성하는 것이 아닙니다. 기능별로 작성하여 그에 맞게 엔드포인트도 작성해주세요
```suggestion
    path('/category/<int:sub_category_id>', ProductListView.as_view())
```

Reply:
아 이해했습니다!!